#! /bin/sh /usr/share/dpatch/dpatch-run
## 14_cap-memsize.dpatch by Thomas Günther <tom@toms-cafe.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add OSD_CAP_MEMSIZE from newer dvb drivers to compile with Debian package
## DP: linux-kernel-headers_2.5.999-test7-bk-17 (Sarge).

@DPATCH@
--- vdr-1.3.37/dvbosd.c
+++ vdr-1.3.37/dvbosd.c
@@ -11,1 +11,9 @@
 #include <linux/dvb/osd.h>
+#ifndef OSD_CAP_MEMSIZE
+typedef struct osd_cap_s {
+        int  cmd;
+#define OSD_CAP_MEMSIZE         1  /* memory size */
+        long val;
+} osd_cap_t;
+#define OSD_GET_CAPABILITY      _IOR('o', 161, osd_cap_t)
+#endif
